lib/raxx/kit.ex

```diff
@@ -78,7 +84,10 @@ defmodule Raxx.Kit do
   defp copy_template(file, root, assigns) do
     case File.read(file) do
       {:error, :eisdir} ->
-        path = Path.relative_to(file, root)
+        path =
+          Path.relative_to(file, root)
+          |> translate_path(assigns)
```

Comment:
Is this the bonus bugfix? :thinking:

Reply:
yeah, we were getting `app_name` directories everywhere. The funny thing is, my previous bug fix unearthed this one, so now I feel a bit guilty ;)

Reply:
so I've just spotted all the app_name dirs in my project. As this fixes it will just ignore for the time being